projector-util-loading/src/main/kotlin/org/jetbrains/projector/util/loading/state/IdeState.kt

```diff
@@ -63,14 +64,20 @@ public enum class IdeState {
     }
 
   public companion object {
-    public val isIdeAttached: Boolean get() = try {
-      WindowsCommandLineProcessor.ourMainRunnerClass
-      true
-    } catch (t: Throwable) {
-      false
-    }
+    public val isIdeAttached: Boolean
+      get() = attachToIde ?: try {
```

Comment:
It will cause exception when no IDE attached
`java.lang.ClassNotFoundException: com.intellij.ide.WindowsCommandLineProcessor`
Is it intended? If not, may be change as follows?

```suggestion
      get() = attachToIde.takeIf { it != true } ?: try {
```

Reply:
Indeed the exception will happen if a user explicitly specified the `true` value before running with a not-ij-based app. Do you think it's a problem and we should provide safety code that you suggest for that situation?

Reply:
I think yes